core/src/main/java/com/novoda/landingstrip/ScrollingPageChangeListener.java

```diff
@@ -4,23 +4,24 @@
 
 class ScrollingPageChangeListener implements ViewPager.OnPageChangeListener {
 
-    private final State state;
-    private final TabsContainer tabsContainer;
+    private final MutableState state;
+    private final TabsContainerView tabsContainerView;
     private final ScrollOffsetCalculator scrollOffsetCalculator;
     private final Scrollable scrollable;
     private final FastForwarder fastForwarder;
-    private final OnPageChangedListenerCollection onPageChangedListenerCollection;
 
     private boolean firstTimeAccessed = true;
 
-    ScrollingPageChangeListener(State state, TabsContainer tabsContainer, ScrollOffsetCalculator scrollOffsetCalculator,
-                                Scrollable scrollable, FastForwarder fastForwarder, OnPageChangedListenerCollection onPageChangedListenerCollection) {
+    ScrollingPageChangeListener(MutableState state,
```

Comment:
Not necessarily. This is a line that you touched anyway, why not change it to better? Improvement can come in small steps 😄